src/test/java/eu/deltasource/internship/hotelmanagementsystem/TestScenario.java

```diff
@@ -0,0 +1,136 @@
+package eu.deltasource.internship.hotelmanagementsystem;
+
+
+import eu.deltasource.internship.hotelmanagementsystem.hotel.service.domain.commodities.*;
+import org.junit.jupiter.api.Test;
+
+
+import java.time.LocalDate;
+import java.util.ArrayList;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+
+
+import static org.hamcrest.CoreMatchers.*;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.core.IsEqual.equalTo;
+
+
+/**
+ * UNDER CONSTRUCTION
+ */
+
+
+public class TestScenario {
+
+
+    @Test
+    public void test() throws NoRoomsAvailableException {
+
+        // given
+
+        // bookings for the first room
+
+        Set<Booking> bookingsFirstRoom = new HashSet<>();
+        bookingsFirstRoom.add(new Booking(9503123552L, LocalDate.of(2019, 5, 17),
```

Comment:
the dates and constants are not really readable, let's extract some costants f.e. 9503123552L -> GUEST_ID
the dates can be extracted as `bookingOneStart = LocalDate.of(2019, 5, 17);` `bookingTwoEnd` etc ...